src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -617,7 +632,7 @@ protected DynamoModel(IStartConfiguration config)
                     }
                     catch (Exception ex)
                     {
-                        Logger.Log(ex.Message);                       
```

Comment:
It's automatic.  I think I installed an extension to do that a long time ago.
Most of the projects I've worked on have "no trailing whitespace" as a project style.